lldb/test/API/functionalities/gdb_remote_client/TestStopPCs.py

```diff
@@ -10,13 +10,15 @@ class TestStopPCs(GDBRemoteTestBase):
     def test(self):
         class MyResponder(MockGDBServerResponder):
             def haltReason(self):
-                return "T02thread:1ff0d;threads:1ff0d,2ff0d;thread-pcs:10001bc00,10002bc00;"
+                return "T02thread:1ff0d;threads:1ff0d,2ff0d,3ff0d;thread-pcs:10001bc00,10002bc00,10003bc00;"
 
             def threadStopInfo(self, threadnum):
                 if threadnum == 0x1FF0D:
```

Comment:
One more comment here please :)

Something like:
# lldb should treat the default halt reason, hwbreak and swbreak in the same way. Which is that it expects the stub to have corrected the PC already, so lldb should not modify it further.

Reply:
Now fixed in https://github.com/llvm/llvm-project/commit/51a356b384f28cce3b2cd874fbba139414b0e207